engine/docker/docker.go

```diff
@@ -254,7 +254,7 @@ func (e *Docker) create(ctx context.Context, pipelineConfig *spec.PipelineConfig
 	}
 
 	_, err := e.client.ContainerCreate(ctx,
-		toConfig(pipelineConfig, step),
+		toConfig(pipelineConfig, step, pipelineConfig.TTY),
```

Comment:
could we do `toConfig(pipelineConfig, step)`?

Reply:
i need better 👓